### DISCUSSION TITLE
【图集】美图分享20

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#1334] [其他]
### 问一下有没有什么图床网站，可以发大图片的那种 GitHub只能10MB 我发个图都得挑挑拣拣
![Random741](https://github.com/user-attachments/assets/2d9123c9-055a-4dae-a941-fee8978c9d3a)
![Random744](https://github.com/user-attachments/assets/dbde59b2-f45e-4e5e-bae3-40ac83d73b5c)
![Random749](https://github.com/user-attachments/assets/2a07ff2c-6c1d-43e4-bea8-323e3f2a3576)
![Random750](https://github.com/user-attachments/assets/8fd6e497-e407-415b-af43-44c54459b603)
![Random757](https://github.com/user-attachments/assets/a086b31e-7e7a-47af-acd9-eb20fadddd59)
![Random759](https://github.com/user-attachments/assets/d0350a9f-a5d2-40c9-b07c-d352e07b900e)
![Random761](https://github.com/user-attachments/assets/1274a752-e62c-45ec-8460-be5b211a7602)
![Random765](https://github.com/user-attachments/assets/44046cd5-ee4c-4c52-9fd1-7b621bfa2f58)
![Random771](https://github.com/user-attachments/assets/2e969a2a-9198-4fe4-9321-85d0a05f9be2)
![Random788](https://github.com/user-attachments/assets/728913bb-184d-4423-9f84-7a44dc614f6f)
![Random789](https://github.com/user-attachments/assets/31975459-b085-438d-810c-4325b366583d)
![Random790](https://github.com/user-attachments/assets/cbb28a76-6001-4ae3-96ac-2d00ab36b37f)
![Random791](https://github.com/user-attachments/assets/c900c834-31c6-4481-9c87-d54913f27bde)
![Random792](https://github.com/user-attachments/assets/c8837b5d-179c-4063-ad8f-7702f30ba5c9)
![Random706](https://github.com/user-attachments/assets/2cde3a2d-3ed7-4e10-992f-8a02a87ade67)
![Random707](https://github.com/user-attachments/assets/47f5c00f-ebeb-48f3-94c5-6d85ab7a04dd)
![Random709](https://github.com/user-attachments/assets/d88aa69d-289c-4ef0-903d-44994cf1c829)
![Random710](https://github.com/user-attachments/assets/72f36773-01a0-4897-bb51-4763de05cea3)
![Random711](https://github.com/user-attachments/assets/21780de2-fe40-440b-8ef7-3697dce2a079)
![Random712](https://github.com/user-attachments/assets/5fd2f0ca-5d89-4e7b-a6b5-dabac45162d3)
![Random713](https://github.com/user-attachments/assets/f0db2a30-37cb-4186-aec4-3c7a5d1fcc08)
![Random714](https://github.com/user-attachments/assets/e212439e-1c91-4abf-a9c9-ac2a746036c4)
![Random716](https://github.com/user-attachments/assets/19d40c0b-8568-4dad-b734-c8d2ba589eb5)
![Random717](https://github.com/user-attachments/assets/0fd22eac-d7a0-47a9-a88a-5686f13a0bd0)
![Random720](https://github.com/user-attachments/assets/116ceb48-3ced-47df-a233-e9e4bd2fe521)
![Random722](https://github.com/user-attachments/assets/94fd2005-d7bf-442c-9fa0-0af214502f52)
![Random723](https://github.com/user-attachments/assets/b403a693-a2cc-4f8b-ac5e-28d1a20fff08)
![Random724](https://github.com/user-attachments/assets/305ccf8c-f558-4bc2-9bec-55e6bc980246)
![Random725](https://github.com/user-attachments/assets/ca95a7f5-96ac-4157-ac85-0b5cd03153c0)
![Random727](https://github.com/user-attachments/assets/b260f7ea-cc86-4a81-8346-bf35e98767a3)
![Random729](https://github.com/user-attachments/assets/c800764b-7ba4-4349-88e9-dde94c4afb73)
![Random731](https://github.com/user-attachments/assets/ac6aa4f6-86f7-4531-8461-56898706bccc)

**Comment by Myitian** [comment]
[freeimage.host](https://freeimage.host/)\
不注册最大 64MB，注册后最大 128MB\
示例：
|15360x13312，26.3MiB(27.6MB)|
|-|
|[https://iili.io/d0FeSaa.png](https://iili.io/d0FeSaa.png)|
|![YSMAP@15%13](https://iili.io/d0FeSaa.png)|
---
如果是 Pixiv 的图片，可以直接用 [pixiv.re](https://pixiv.re/)/pixiv.nl/pixiv.cat 图片反向代理\
示例：
|单张模式|多张模式|
|-|-|
|`https://pixiv.re/<ID>.jpg`|`https://pixiv.re/<ID>-<从1开始的序号>.jpg`|
|[https://pixiv.re/106345139.jpg](https://pixiv.re/106345139.jpg)|[https://pixiv.re/85633671-1.jpg](https://pixiv.re/85633671-1.jpg)|
|![归终](https://pixiv.re/106345139.jpg)|![刻晴](https://pixiv.re/85633671-1.jpg)|

**Comment by Buer-Nahida** [comment]
1. 可以新建个github仓库上传图片（不过你要学习git命令行的使用方式，否则只能上传25M以下的文件），然后通过
https:\/\/raw.githubusercontent.com\/***你的Github用户名***/***你的Github仓库名***/***你的分支名（默认为main或master）***/***你的文件路径（使用正斜杠分割）***
超过100MB后需要使用Git LFS(Large File Storage)上传，单个文件超过1GB就没办法了，只能开通Github Pro，我这么干过——https://github.com/Buer-Nahida/rawfiles
2. 可以用vercel自己搭个网站，然后把图片上传上去，我现在是这么干的——[naxida.me](https://naxida.me)

**Reply by capacitor1** [reply]
🤔考虑使用GitHub仓库，如果文件过大可以考虑把文件切两半doge

不过建网站这事，我自己是建有网站但是。。全是~~涩涩~~，不方便公开

**Reply by Myitian** [reply]
> 可以新建个github仓库上传图片（不过你要学习git命令行的使用方式，否则只能上传10M以下的文件）

> 超过45MB后需要使用Git LFS(Large File Storage)上传，单个文件超过1GB就没办法了，只能开通Github Pro

不是哦，见[关于 GitHub 上的大文件 - GitHub 文档](https://docs.github.com/zh/repositories/working-with-files/managing-large-files/about-large-files-on-github)，网页限制是 25 MiB ，无 LFS 时 Git 推送限制是 100 MiB

> GitHub 限制存储库中允许的文件大小。 如果尝试添加或更新大于 50 MiB 的文件，您将从 Git 收到警告。 更改仍将成功推送到仓库，但您可以考虑删除提交，以尽量减少对性能的影响。 有关详细信息，请参阅“[从存储库的历史记录中删除文件](https://docs.github.com/zh/repositories/working-with-files/managing-large-files/about-large-files-on-github#removing-files-from-a-repositorys-history)”。
> 
> > 注意：如果通过浏览器将文件添加到存储库，该文件不得大于 25 MiB。 有关详细信息，请参阅“[添加文件到仓库](https://docs.github.com/zh/repositories/working-with-files/managing-files/adding-a-file-to-a-repository)”。
> 
> GitHub 阻止大小超过 100 MiB 的文件。
> 
> 要跟踪超出此限制的文件，必须使用 Git Large File Storage (Git LFS)。 有关详细信息，请参阅“[关于 Git Large File Storage](https://docs.github.com/zh/repositories/working-with-files/managing-large-files/about-git-large-file-storage)”。

另外 Git LFS 的带宽和容量是计费的，每月免费额度 1 GiB ，见[关于 Git Large File Storage 的计费 - GitHub 文档](https://docs.github.com/zh/billing/managing-billing-for-git-large-file-storage/about-billing-for-git-large-file-storage)和[关于 GitHub 的计费 - GitHub 文档](https://docs.github.com/zh/billing/managing-your-github-billing-settings/about-billing-on-github#included-amounts-by-plan)

> 每个使用 Git Large File Storage 的帐户都会获得 1 GiB 的免费存储空间和一个月的免费带宽 1 GiB。 如果带宽和存储配额不够，您可以选择为 Git LFS 购买额外配额。 未使用的带宽不会结转到下个月。

> <table><tbody><tr><th><b>产品</b></th><th><b>使用情况</b></th><th><b>GitHub Free</b></th><th><b>GitHub Pro</b></th><th><b>组织的 GitHub Free</b></th><th><b>GitHub Team</b></th><th>GitHub Enterprise Cloud</th></tr><tr><th colspan="7">……</th></tr><tr><th rowspan="2"><b>Git Large File Storage</b></th><th><b>存储空间（每月）</b></th><td>1 GB</td><td>1 GB</td><td>1 GB</td><td>1 GB</td><td>1 GB</td></tr><tr><th><b>带宽（每月）</b></th><td>1 GB</td><td>1 GB</td><td>1 GB</td><td>1 GB</td><td>1 GB</td></tr></tbody></table>

**Reply by Buer-Nahida** [reply]
> > 可以新建个github仓库上传图片（不过你要学习git命令行的使用方式，否则只能上传10M以下的文件）
> 
> > 超过45MB后需要使用Git LFS(Large File Storage)上传，单个文件超过1GB就没办法了，只能开通Github Pro
> 
> 不是哦，见[关于 GitHub 上的大文件 - GitHub 文档](https://docs.github.com/zh/repositories/working-with-files/managing-large-files/about-large-files-on-github)，网页限制是 25 MiB ，无 LFS 时 Git 推送限制是 100 MiB
> 
> > GitHub 限制存储库中允许的文件大小。 如果尝试添加或更新大于 50 MiB 的文件，您将从 Git 收到警告。 更改仍将成功推送到仓库，但您可以考虑删除提交，以尽量减少对性能的影响。 有关详细信息，请参阅“[从存储库的历史记录中删除文件](https://docs.github.com/zh/repositories/working-with-files/managing-large-files/about-large-files-on-github#removing-files-from-a-repositorys-history)”。
> > > 注意：如果通过浏览器将文件添加到存储库，该文件不得大于 25 MiB。 有关详细信息，请参阅“[添加文件到仓库](https://docs.github.com/zh/repositories/working-with-files/managing-files/adding-a-file-to-a-repository)”。
> > 
> > 
> > GitHub 阻止大小超过 100 MiB 的文件。
> > 要跟踪超出此限制的文件，必须使用 Git Large File Storage (Git LFS)。 有关详细信息，请参阅“[关于 Git Large File Storage](https://docs.github.com/zh/repositories/working-with-files/managing-large-files/about-git-large-file-storage)”。
> 
> 另外 Git LFS 的带宽和容量是计费的，每月免费额度 1 GiB ，见[关于 Git Large File Storage 的计费 - GitHub 文档](https://docs.github.com/zh/billing/managing-billing-for-git-large-file-storage/about-billing-for-git-large-file-storage)和[关于 GitHub 的计费 - GitHub 文档](https://docs.github.com/zh/billing/managing-your-github-billing-settings/about-billing-on-github#included-amounts-by-plan)
> 
> > 每个使用 Git Large File Storage 的帐户都会获得 1 GiB 的免费存储空间和一个月的免费带宽 1 GiB。 如果带宽和存储配额不够，您可以选择为 Git LFS 购买额外配额。 未使用的带宽不会结转到下个月。
> 
> > 产品使用情况GitHub FreeGitHub Pro组织的 GitHub FreeGitHub TeamGitHub Enterprise Cloud……Git Large File Storage存储空间（每月）1 GB1 GB1 GB1 GB1 GB带宽（每月）1 GB1 GB1 GB1 GB1 GB

啊？但我上次推68MB的文件就给我报错说文件太大，还是上了git lfs才传上去的

**Comment by hsc515** [comment]
豪堪

**Comment by shenguojia123** [comment]
我觉得可以把图片切成几条发上来（）

**Comment by Kuuga2000skills** [comment]
114514观看记录

**Reply by Kuuga2000skills** [reply]
等等，小丑了，原来是彩蛋。哼啊啊啊啊！

**Comment by Myitian** [comment]
似乎GitHub会把非GitHub来源的图片全部转到`camo.githubusercontent.com`域名下，所以好像根本放不了图床上的大图，建议还是自己建一个GitHub仓库当图床，比如这个图就是在仓库里的（网页最大25MiB，无LFS时Git推送最大100MiB）：
![YSMAP v4](https://raw.githubusercontent.com/Myitian/-01/main/YSMAP.4.png)

**Comment by keyuanya** [comment]
https://imagestool.com/zh_CN/
有一个麻烦点的办法，用这个网站转换一下图片格式，建议JPG格式质量95%，质量和文件大小都比较平衡

**Reply by capacitor1** [reply]
🤔不赞同，因为分享图片要求原图 无损

**Reply by uid10644088** [reply]
赞同

**Comment by Vilei2000** [comment]
什么gayhub，这里是绳网！（KY乱入）
